CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+### citus v12.1.5 (July 7, 2024) ###
```

Comment:
Kindly request to change the date into July 10

Reply:
I will end up updating the date again as we need to wait until one more fix is complete.